index.html

```diff
@@ -333,23 +333,203 @@ <h1>Icon Buttons</h1>
+  
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+     
+      
     </div>
     
-</div>
+    
+    
+    <div class="container">
+      <h1>Vertical cards</h1>
+      <p>Here we created cards.</p>
+      <p>
+        The card component with simple .
+      </p>
+      <div class="vertical-card">
+        <div class="card-simple">
+          <div class="card__container">
+            <p class="card__body">
+              <img src="https://m.media-amazon.com/images/I/71P5lELL6lL._AC_SR180,120_QL70_.jpg" alt="">
+           <h4>Speed Hybrid Bicyle <i class="far fa-heart"></i><p><span class="price-discount">&#8377 16499 </span><span class="price-originl">&#8377 17000</span> <span class="offer">(3%OFF)</span> </p> </h4>
+           <span>Save extra with No Cost EMI</span>
+           <div class="cart-button"><button><i class="fas fa-shopping-cart"></i>Add to cart</button></div>
+          </p>  
+          </div>
+        </div>
+        </div>
+      </div>
 
+      </div>
 
+    <div class="code-container">
+      <iframe
```

Comment:
customize the iframe so that it will have same width and height throughout the doc, it makes ui good